components/dashboards-web-component/src/designer/DashboardCreatePage.jsx

```diff
@@ -17,50 +17,158 @@
  *
  */
 
-import React, {Component} from 'react';
-import {FormattedMessage} from 'react-intl';
-import {MuiThemeProvider, darkBaseTheme, getMuiTheme} from 'material-ui/styles';
-import {RaisedButton, TextField, Snackbar,} from 'material-ui';
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+import { MuiThemeProvider, darkBaseTheme, getMuiTheme } from 'material-ui/styles';
+import { RaisedButton, TextField, Snackbar } from 'material-ui';
 import ClearIcon from 'material-ui/svg-icons/content/clear';
 
-import {FormPanel, Header} from '../common';
+import { FormPanel, Header } from '../common';
 import DashboardAPI from '../utils/apis/DashboardAPI';
 import DashboardUtils from '../utils/DashboardUtils';
+import { HttpStatus } from '../utils/Constants';
 
+/**
+ * Material UI theme.
+ */
 const muiTheme = getMuiTheme(darkBaseTheme);
-const styles = {
-    fieldError: {color: "#FF5722"},
-    messageBox: {textAlign: "center", color: "white"},
-    errorMessage: {backgroundColor: "#FF5722", color: "white"},
-    successMessage: {backgroundColor: "#4CAF50", color: "white"}
-};
 
-const httpStatus = {
-    accepted: 201,
-    conflict: 409
+/**
+ * Style constants.
+ */
+const styles = {
+    fieldError: { color: '#FF5722' },
+    messageBox: { textAlign: 'center', color: 'white' },
+    errorMessage: { backgroundColor: '#FF5722', color: 'white' },
+    successMessage: { backgroundColor: '#4CAF50', color: 'white' },
 };
 
+/**
+ * Create dashboard component.
+ */
 export default class DashboardCreatePage extends Component {
+    /**
+     * Constructor.
+     */
     constructor() {
         super();
         this.state = {
             showMessage: false,
             message: '',
             url: '',
             fieldErrors: {
-                name: {error: '', style: ''},
-                url: {error: '', style: ''}
+                name: { error: '', style: '' },
```

Comment:
Extra spaces have been added here

Reply:
@lasanthaS Is this from es-lint as well?

Reply:
This is enforced by ESLint